index.html

```diff
@@ -0,0 +1,308 @@
+<!doctype html>
+
+<html lang="en">
+	<head>
+		
+  		<title>My Portfolio</title>
+  		
+  		<link rel = "stylesheet" type = "text/css" href = "css/styles.css" />
+  		<link rel = "stylesheet" type = "text/css" href = "css/normalize.css" />
+
+	</head>
+	
+	<body>
+		<nav>
+			<div class="navSpacer">
+				<div>	
+				</div>
+			</div>
+				
+			<div class="navContainer">
+				<div class="navDot">
+				</div>
+				
+				<div class="logoBox">
+					<a href="http://google.com" target="_blank" class="logoLink">
```

Comment:
best practice is not to put block level elements inside inline elements. a tag is inline and p tag is block so swap the order and use css to style the a tag to take up 100% height and width.